addons/path-mapped/common/src/main/java/org/commonjava/indy/pathmapped/cache/PathMappedMavenGACache.java

```diff
@@ -232,12 +235,19 @@ public void fill()
         Set<String> scanned = getScannedStores();
 
         // find not-scanned stores, notScanned = matched - scanned
-        Set<String> notScanned = new HashSet<>( matched );
+        List<String> notScanned = new ArrayList<>( matched );
         notScanned.removeAll( scanned );
 
         if ( !notScanned.isEmpty() )
         {
-            scanAndUpdate( notScanned );
+            List<List<String>> batches = Lists.partition( notScanned, 100 );
```

Comment:
Magic number to constant? Or should it be configurable?

Reply:
Yeah, I will use a constant then. I guess configurable is a bit too much for this.